Automattic/wp-calypso
# Jetpack Manage: Preview Pane: Re-make the Preview Pane header to match A4A Figma

Resolve: https://github.com/Automattic/jetpack-manage/issues/326

## Proposed Changes

* Update the preview pane according to the new Figma design.

## Testing Instructions

- Open the new sites listing ( http://jetpack.cloud.localhost:3000/sites )
- Click on any site
- The new preview panel should open
- Check the updated header according to https://github.com/Automattic/jetpack-manage/issues/326
  - The `X` (close button) should be functional


## Pre-merge Checklist
